responsibleai/responsibleai/_internal/constants.py

```diff
@@ -19,6 +19,19 @@ class Metadata(object):
 
     META_JSON = 'meta.json'
     MODEL = 'model'
+    TRAIN = 'train'
+    TEST = 'test'
+    TASK_TYPE = 'task_type'
+    TARGET_COLUMN = 'target_column'
+    CLASSES = 'classes'
+    FEATURE_COLUMNS = 'feature_columns'
+    FEATURE_METADATA = 'feature_metadata'
+    FEATURE_RANGES = 'feature_ranges'
+    CATEGORICAL_FEATURES = 'categorical_features'
```

Comment:
I wonder if some of these should go in a more common place like raiutils, maybe not in this PR.... they seem to be re-used in error analysis and other packages.

Reply:
Love this idea! I agree that it should happen separately, though. Let me note that...